src/properties/degrees.c

```diff
@@ -517,3 +517,139 @@ igraph_error_t igraph_sort_vertex_ids_by_degree(const igraph_t *graph,
     }
     return IGRAPH_SUCCESS;
 }
+
+/**
+ * \function igraph_construct_jdm
+ * \brief Constructs a joint degree matrix.
+ *
+ * The joint degree matrix of a graph contains the number of edges, or the sum of the weights, between vertices of
+ * degree i and degree j for every (i, j). The function populates a joint degree matrix and works on directed
+ * and undirected graphs. Self loops are only counted once. If either \p dout or \p din are <= 0, the JDM
```

Comment:
```suggestion
 * and undirected graphs. Self loops are only counted once. If either \p dout or \p din are &lt;= 0, the JDM
```

To fix the doc build. (I thought you were on strike today.)

Reply:
Unfortunately, my course deadlines wouldn't strike with me 😆